dotcom-rendering/src/web/lib/contributions.ts

```diff
@@ -127,7 +127,7 @@ const dateDiffDays = (from: number, to: number): number => {
 
 const AskPauseDays = 90;
 
-export const isRecentOneOffContributor = () => {
+export const isRecentOneOffContributor = (): boolean => {
```

Comment:
This just makes sense to have this type here. You can infer it from the name but it's nice that it's super clear